PORT-11428 Added mapping extra resources page to octopus deploy guide (#1824)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/cicd/octopus-deploy/mapping-extra-resources/mapping-extra-resources.md

> ---

Reviewer: in addition to all other comments, add a link to this page in the main "octopus deploy" page, with something like:
"The integration supports additional resources, see the `[ingest additional resources]()` page for more information:".

---
import RunbookBlueprint from './blueprints/\_runbook.mdx'
import RunbookMapping from './mappings/\_runbook.mdx'
import TaskBlueprint from './blueprints/\_task.mdx'
import TaskMapping from './mappings/\_task.mdx'
import TeamBlueprint from './blueprints/\_team.mdx'
import TeamMapping from './mappings/\_team.mdx'
import WorkePoolBlueprint from './blueprints/\_workerpool.mdx'
import WorkerPoolMapping from './mappings/\_workerpool.mdx'
import WorkerBlueprint from './blueprints/\_worker.mdx'
import WorkerMapping from './mappings/\_worker.mdx'
import EnvironmentBlueprint from './blueprints/\_environment.mdx'
import EnvironmentMapping from './mappings/\_environment.mdx'
import ProxyBlueprint from './blueprints/\_proxy.mdx'
import ProxyMapping from './mappings/\_proxy.mdx'
import AccountBlueprint from './blueprints/\_account.mdx'
import AccountMapping from './mappings/\_account.mdx'
import PackageBlueprint from './blueprints/\_package.mdx'
import PackageMapping from './mappings/\_package.mdx'
import SubscriptionBlueprint from './blueprints/\_subscription.mdx'
import SubscriptionMapping from './mappings/\_subscription.mdx'

# Mapping Extra Resources for Octopus Deploy Integration

> As you've probably looked at some examples on [Octopus Deploy](../octopus-deploy.md) page, you've noticed that the Octopus Deploy integration supports several resources, but there are more resources that are not documented on the page.

Reviewer: ```suggestion
In addition to the supported resources listed in the [integration page](../octopus-deploy.md), other resources can be ingested, as described on this page.
```

> As you've probably looked at some examples on [Octopus Deploy](../octopus-deploy.md) page, you've noticed that the Octopus Deploy integration supports several resources, but there are more resources that are not documented on the page.

Reviewer: in addition to the suggestion I left, please change the link to be full path and not relative

> As you've probably looked at some examples on [Octopus Deploy](../octopus-deploy.md) page, you've noticed that the Octopus Deploy Integration supports several resources, but there are many more resources that are not documented on the page.

Reviewer 2: ```suggestion
As you've probably looked at some examples on [Octopus Deploy](../octopus-deploy.md) page, you've noticed that the Octopus Deploy integration supports several resources, but there are more resources that are not documented on the page.
```

This page will help you understand what kind of Octopus Deploy resources are supported by the integration and how to map them into Port.

> ## Is the Resource Supported by the Octopus Deploy Integration?

Reviewer: ```suggestion
## Check if a resource is supported
```

ged
### Mapping the Resource to Port

After determining that the resource is supported by the API, you can map it to Port by following these steps:
- **Define the resource kind**: The value of `kind` should be the value of the resource in the api pattern **`GET /{spaceId}/{resources}`** without the trailing `s`. For example, if API path is **`GET /{spaceId}/runbooks`**, the value of `kind` should be `runbook`.
- **Define the selector**: The selector is a JQ query that filters the resources you want to ingest. If the query evaluates to `false`, the resource will be skipped.
- **Define the properties**: The properties are the fields of the Port entity. You can map the resource fields to the entity properties using JQ queries.
- **Define the blueprint**: The blueprint is the name of the Port entity blueprint that you want to use for the resource.
- **Define the relations**: You can define `relations` between the resource and other resources in Port by mapping the related fields.

> ## Example Configuration for Additional Resources

Reviewer: ```suggestion
## Example configurations
```
